cmd/rook/ceph/osd.go

```diff
@@ -206,21 +206,21 @@ func writeOSDConfig(cmd *cobra.Command, args []string) error {
 
 	clientset, _, _, err := rook.GetClientset()
 	if err != nil {
-		rook.TerminateFatal(fmt.Errorf("failed to init k8s client. %+v\n", err))
+		rook.TerminateFatal(fmt.Errorf("failed to init k8s client %+v", err))
 	}
 
 	context := createContext()
 	context.Clientset = clientset
 	commonOSDInit(osdConfigCmd)
 	locArgs, err := client.FormatLocation(cfg.location, cfg.nodeName)
 	if err != nil {
-		rook.TerminateFatal(fmt.Errorf("invalid location %s. %+v\n", cfg.location, err))
+		rook.TerminateFatal(fmt.Errorf("invalid location %s %+v", cfg.location, err))
```

Comment:
what is the reason for the periods being removed that separate the first sentence from the error message?

Reply:
To correct linter errors @jbw976